### PR TITLE
student audit sample queries

### PR DESCRIPTION
**Pull request by chdietz**
Added samples for student and student_ethnicity tables

### CODE REVIEW

File: docs/Audit.md [hunk 1]
```diff
@@ -355,3 +359,244 @@ ORDER BY acc_audit.exam_id, acc_audit.action_date DESC;
...
+|          1 | SSID001 | Gladys     | Ruth        | Durrant         |     15 | dwtest@example.com | 2017-11-10 09:24:11.604717 | update   | 2001-06-23 | Llama Sabrewing HS         |
+|          1 | SSID001 | Gladys     | Ruth        | Durrant         |     13 | dwtest@example.com | 2017-11-10 09:23:01.498262 | update   | 2001-06-23 | Llama Sabrewing HS         |
+|          1 | SSID001 | Gladys     |             | Durrant         |     12 | dwtest@example.com | 2017-11-10 09:21:51.427089 | update   | 2001-06-23 | Llama Sabrewing HS         |
+|          1 | SSID001 | Gladys     |             | Durrant         |     10 | dwtest@example.com | 2017-11-10 09:20:41.278709 | update   | 2001-06-23 | Llama Sabrewing HS         |
+|          1 | SSID001 | Gladys     |             | Durrant         |      6 | dwtest@example.com | 2017-11-10 09:19:31.054923 | original | 2001-06-23 | Llama Sabrewing HS         |
++------------+---------+------------+-------------+-----------------+--------+--------------------+----------------------------+----------+------------+----------------------------+
+7 rows in set (0.00 sec)
```
Comment (chdietz) [inline docs/Audit.md]: I will regenerate output on the queries once the inferred school changes are complete to see if this still looks the same

File: docs/Audit.md [hunk 2]
```diff
@@ -355,3 +359,244 @@ ORDER BY acc_audit.exam_id, acc_audit.action_date DESC;
 9 rows in set (0.00 sec)
 ```
 
+#### Student
+
+**Finding modified students**
+The following query outputs one row for each modified student.  It includes the count of modifications and the date of the last change.
+
+The where clause can be changed to include different sets students.
```
Comment (jtreuting) [inline docs/Audit.md]: nit: sets of students

File: docs/Audit.md [hunk 3]
```diff
@@ -355,3 +359,244 @@ ORDER BY acc_audit.exam_id, acc_audit.action_date DESC;
...
+| SSID003 | Mark       |             | Anderson        |                    1 | 2017-11-10 09:25:21.846849 |
++---------+------------+-------------+-----------------+----------------------+----------------------------+
+3 rows in set (0.00 sec)
+```
+
+**Finding modifications to students**
+The following query outputs one row for each student.  It includes the count of modifications and the date of the last change.
+Students with no modifications have a `student_update_count` of `0`.
+
+The where clause can be changed to include different sets students.
```
Comment (jtreuting) [inline docs/Audit.md]: same

File: docs/Audit.md (hunk 2 above)
Comment (chdietz) [reply docs/Audit.md]: sets is confusing, can just drop `sets`

Comment (malaffoon) [reply docs/Audit.md]: Clearly, the user needs to watch @agorina's talk on "Thinking in Sets"

Comment (chdietz) [reply docs/Audit.md]: yes, and read the book.

**Pull request merged**